portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,8 +43,36 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     int randomIndex = (int)Math.floor(Math.random() * ((int)jsonString.size()));
     System.out.print(jsonString.size() + " randome num: " + randomIndex);
     response.getWriter().println(jsonString.get(randomIndex).toString());
+  }
+
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = getParameter(request, "text-input", "");
+    boolean upperCase = Boolean.parseBoolean(getParameter(request, "upper-case", "false"));
+    
+    // Convert the text to upper case.
+    if (upperCase) {
+      text = text.toUpperCase();
+    }
 
+    //@TODO color still need to be implemented 
 
+    // Respond with the result.
+    response.setContentType("text/html;");
+    response.getWriter().println("Thanks for your comment: \n\"" + text + "\"\n Hope you have a good day!");
+    
+  }
+
+  /**
+   * @return the request parameter, or the default value if the parameter
+   *         was not specified by the client
+   */
+  private String getParameter(HttpServletRequest request, String name, String defaultValue) {
```

Comment:
Can this be a static function?

Reply:
I'm not sure sorry.

Reply:
this function does not use any state from this class and can be made static. https://www.edureka.co/blog/java-static-method/